File: src/app/Models/Job.ts
import {JobsFields  } from "./JobsFields";
export class Job{
    id:number|null=null
    jobField:JobsFields|null = null
    jobName:string|null = null
    numHours:number|null = null
    jobArea:string|null = null
    requirements:string|null = null
    homeWorking:boolean|null = null
}

File: Server/WebApplication1/Models/job.cs
namespace WebApplication1.Models

{
    public class Job
    {public int Id{ get; set; }
        public JobsFields JobField { get; set; }
        public string JobName { get; set; }
        public int NumHours { get; set; }
        public string JobArea { get; set; }
        public string Requirements { get; set; }
         public bool HomeWorking { get; set; }

    }

    public enum JobsFields
    {
        Software_Development,
        Healthcare,
        Finance,
        Hospitality_and_Tourism,
        Retail_and_Sales,
        Human_Resources,
        Engineering,
            }
}
